ui/scss/component/_markdown-preview.scss

```diff
@@ -13,7 +13,33 @@
     font-size: inherit;
     font-weight: var(--font-weight-bold);
     margin-bottom: var(--spacing-medium);
-    padding-top: var(--spacing-medium);
+
+    &:not(:first-child) {
+      margin-top: var(--spacing-large);
+    }
+  }
+
+  h1 {
+    font-size: 1.8em;
```

Comment:
Not sure of the best way to do this

Reply:
Should an h1 in markdown be as big as the file title? Or a little smaller?

Reply:
I'm torn on this. Probably smaller.

Reply:
What do other blogging platforms do? Does Medium all multiple h1s? Are they smaller than title?